README.md

```diff
@@ -8,24 +8,44 @@
 
 ## About
 
-This is the Git repo of the official Helm charts for YOURLS.
+This is the Git repository of the official Helm charts for YOURLS.
```

Comment:
Not sure to understand this change.

Reply:
This was because of https://github.com/YOURLS/charts/runs/5355453994?check_suite_focus=true#step:4:135

Reply:
Oh ok, I see, thanks